bindings/python/pymongoarrow/api.py

```diff
@@ -233,3 +239,42 @@ def aggregate_numpy_all(collection, pipeline, *, schema, **kwargs):
     return _arrow_to_numpy(
         aggregate_arrow_all(collection, pipeline, schema=schema, **kwargs), schema
     )
+
+
+def write(collection, tabular, mode="insert"):
+    validate_schema(tabular.schema)
+    cur_batch = []
+    cur_size = 0
+    cur_offset = 0
+    from collections import Counter
+
+    results = Counter({})
+
+    def transform_bwe(bwe, offset):
```

Comment:
Let's make this a private top-level helper function.

Reply:
Done.